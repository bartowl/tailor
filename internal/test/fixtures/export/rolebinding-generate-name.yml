apiVersion: template.openshift.io/v1
kind: Template
metadata:
  creationTimestamp: null
  name: tailor
objects:
- apiVersion: authorization.openshift.io/v1
  groupNames: null
  kind: RoleBinding
  metadata:
    creationTimestamp: null
    generateName: system:image-pusher-
  roleRef:
    name: system:image-pusher
  subjects:
  - kind: ServiceAccount
    name: default
    namespace: foo-dev
  userNames:
  - system:serviceaccount:foo-dev:default
