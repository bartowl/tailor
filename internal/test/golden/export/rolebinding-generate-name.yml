apiVersion: template.openshift.io/v1
kind: Template
objects:
- apiVersion: authorization.openshift.io/v1
  kind: RoleBinding
  metadata:
    generateName: system:image-pusher-
  roleRef:
    name: system:image-pusher
  subjects:
  - kind: ServiceAccount
    name: default
    namespace: foo-dev
